android/quest/src/main/java/org/smartregister/fhircore/quest/ui/usersetting/UserSettingViewModel.kt

```diff
@@ -228,4 +239,63 @@ constructor(
   suspend fun emitSnackBarState(snackBarMessageConfig: SnackBarMessageConfig) {
     _snackBarStateFlow.emit(snackBarMessageConfig)
   }
+
+  private fun copyDatabase(context: Context, onCopyCompleteListener: () -> Unit) {
+    viewModelScope.launch(dispatcherProvider.io()) {
+      try {
+        val passphrase = DBUtils.getEncryptionPassphrase("fhirEngineDbPassphrase")
```

Comment:
Extract this string `fhirEngineDbPassphrase` to a constant. How are we using this string?